content/en/tutorials/replicate-aws-resources-localstack-extension/index.md

```diff
@@ -0,0 +1,232 @@
+
+```bash 
+{
+    "MD5OfMessageBody": "99914b932bd37a50b983c5e7c90ae93b",
+    "MessageId": "64e8297c-f0b2-4b68-a482-6cd3317f5096"
+}
+```
+
+In the LocalStack logs, you'll see confirmation of the Lambda function invocation along with any debug messages.
+
+```bash 
+
+2024-03-26T07:23:47.842 DEBUG --- [5119b27cdf1e] l.s.l.i.version_manager    : [func1-381c6f7c-3ad8-4c79-aad8-5119b27cdf1e] START RequestId: 381c6f7c-3ad8-4c79-aad8-5119b27cdf1e Version: $LATEST
+2024-03-26T07:23:47.842 DEBUG --- [5119b27cdf1e] l.s.l.i.version_manager    : [func1-381c6f7c-3ad8-4c79-aad8-5119b27cdf1e] Debug output from Lambda function
+2024-03-26T07:23:47.842 DEBUG --- [5119b27cdf1e] l.s.l.i.version_manager    : [func1-381c6f7c-3ad8-4c79-aad8-5119b27cdf1e] END RequestId: 381c6f7c-3ad8-4c79-aad8-5119b27cdf1e
+```
+
+## Run the Replicator extension
+
+To run the Replicator extension:
+
+
+-   Access [`https://aws-replicator.localhost.localstack.cloud:4566`](https://aws-replicator.localhost.localstack.cloud:4566/) via your web browser.
+-   Provide your AWS Credentials: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and optionally `AWS_SESSION_TOKEN`.
+- Add a new YAML-based Proxy configuration to proxy requests for specific resources to AWS. For this scenario, configure it to proxy requests for the SQS queue created earlier.
+  ```yaml 
+  services:
+    sqs:
+      resources:
+        - '.*:test-local-proxy'
+   ```
+- Save the configuration to enable the Replicator extension. Once enabled, you'll see the proxy status as **Enabled**.
+
+To invoke the local Lambda function with the remote SQS queue:
+
+-   Navigate to your AWS Management Console and access **Simple Queue Service**.
+-   Select the **test-local-proxy** queue.
+-   Send a message with a body (e.g., `Hello LocalStack`) by clicking **Send Message**.
+
+You'll observe the local Lambda function being invoked once again, with corresponding debug messages visible in the logs.
+
+```bash 
+2024-03-26T07:45:16.524 DEBUG --- [db58fad602e5] l.s.l.i.version_manager    : [func1-ed938bb0-e1ee-41fb-a844-db58fad602e5] START RequestId: ed938bb0-e1ee-41fb-a844-db58fad602e5 Version: $LATEST
+2024-03-26T07:45:16.524 DEBUG --- [db58fad602e5] l.s.l.i.version_manager    : [func1-ed938bb0-e1ee-41fb-a844-db58fad602e5] Debug output from Lambda function
+2024-03-26T07:45:16.524 DEBUG --- [db58fad602e5] l.s.l.i.version_manager    : [func1-ed938bb0-e1ee-41fb-a844-db58fad602e5] END RequestId: ed938bb0-e1ee-41fb-a844-db58fad602e5
+```
+
+You can even run the standard `awslocal` commands in your terminal that would query the remote cloud resources, instead of the local ones.
+
+Upon completion, you can click **Disable** on the Replicator extension web interface to deactivate the proxy configuration. Additionally, you can delete the remote SQS queue to avoid AWS billing for long-running resources. To remove local resources, stop the LocalStack container to clear the local Lambda function and SQS queue.
```

Comment:
```suggestion
Upon completion, you can click **Disable** on the Replicator extension web interface to deactivate the proxy configuration.
Additionally, you can delete the remote SQS queue to avoid AWS billing for long-running resources.
To remove local resources, stop the LocalStack container to clear the local Lambda function and SQS queue.
```